esm/interface/document.js

```diff
@@ -30,6 +30,7 @@ import {InputEvent} from './input-event.js';
 import {ImageClass} from './image.js';
 import {MutationObserverClass} from './mutation-observer.js';
 import {NodeList} from './node-list.js';
+import {NamedNodeMap} from './named-node-map.js';
```

Comment:
Do you mind moving this above the previous line? Just to preserve the alphabetical order, thanks!

Reply:
Of course, thanks for your feedback 👍

Reply:
Done